### PR TITLE
update install-kubeadm

### PR DESCRIPTION
**Pull request by inductor**
Closes https://github.com/kubernetes/website/issues/22593

### CODE REVIEW

File: content/ja/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md [hunk 1]
```diff
@@ -79,20 +79,20 @@ update-alternatives --set iptables /usr/sbin/iptables-legacy
...
 ### ワーカーノード
 
-| プロトコル | 通信の向き | ポート範囲   | 目的                    | 使用者                  |
-|-----------|------------|-------------|-------------------------|-------------------------|
-| TCP       | Inbound    | 10250       | Kubelet API             | Self, Control plane     |
-| TCP       | Inbound    | 30000-32767 | NodePort Services**     | All                     |
+| プロトコル | 通信の向き | ポート範囲   | 目的                    | 使用者                    |
+|-----------|------------|-------------|-------------------------|---------------------------|
+| TCP       | Inbound    | 10250       | Kubelet API             | 自身、コントロールプレーン |
+| TCP       | Inbound    | 30000-32767 | NodePort Services†     | 全て                      |
```
Comment (sftim) [inline content/ja/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:93]: perhaps even:
```suggestion
| TCP       | Inbound    | 30000-32767 | NodePortのService†     | 全て                      |
```
(or similar)
?

Comment (nasa9084) [reply content/ja/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:93]: +1

**Comment by sftim** [comment]
Thanks @inductor

**Pull request merged**